testScripts/createAKSFile.md

```diff
@@ -0,0 +1,214 @@
+
+# Welcome to this test document
+
+## **FIXED In latest PR with the addition of the remove leading whitespace function** 
```

Comment:
Huh? if it is fixed why is it still here? Is this a note for the reviewer that it is fixed in a PR submitted after this one? If so then make this clearer, preferably with a PR #